website/integrations/services/apache-guacamole/index.mdx

```diff
@@ -67,3 +68,10 @@ openid-redirect-uri=https://guacamole.company/ # This must match the redirect UR
 
   </TabItem>
 </Tabs>
+
+## Guacamole User Creation
+
+In Guacamole - if everything has configured correctly you will see in the bottom left corner `Sign in with: OpenID`
```

Comment:
This button only shows up when multiple authentication plugins are configured, when only OIDC is configured the redirect happens directly